docs/SCD2-sparkcompute.md

```diff
@@ -39,7 +39,9 @@ After filling in null, the resulting records will be:
 | 1  | 100        | John | abc@abc.com   |
 
 **Deduplicate:** Deduplicate consecutive records that have no changes. This can be used with the blacklist to exclude some fields from comparison.
-The blacklist typically includes the start time field. For example, suppose the following records are processed with start_date as blacklist:
+The blacklist typically includes the start time field. If Perserve Target is set to false, the record with the latest record will be picked.
+The end time of a record will be computed as the start time of the next non-deduplicated record minus the offset.
```

Comment:
end time is mentioned here, but is not in the example below. Can you add it so that it is more clear